src/js/components/UploadForm.jsx

```diff
@@ -26,25 +30,52 @@ export const renderErrors = (errors) => {
 }
 
 export const getDropzoneText = ({ code, errors, file }) => {
-  let message = 'Drag your LAR file into this area, or click in this box to select a LAR file to upload.'
-  let fileName = null
+  let howToMessage = 'To select a file to upload, drag it into this box or click here.'
+  if(code >= CREATED) {
+    howToMessage = 'To select a new file to upload, drag it into this box or click here.'
+  }
+  let message = <p>{howToMessage}</p>
 
   if(code >= UPLOADING) {
-    message = 'Drag another LAR file to this area, or click in the box to select a LAR file to upload.'
+    message = howToMessage
+  }
+
+  if(code === SIGNED) {
+    message = <article>
+      <p>Your submission is complete.</p>
+      <p className="file-selected">{howToMessage}</p>
+    </article>
   }
 
   if(file) {
-    message = `${file.name} is ready for upload.`
+    message = <article>
+      <p><strong>{file.name}</strong> selected.</p>
+      <p className="file-selected">{howToMessage}</p>
+    </article>
+
     if(errors.length > 0) {
-      message = `${file.name} can not be uploaded.`
+      message = <article>
+        <p><strong>{file.name}</strong> can not be uploaded.</p>
+        <p>{howToMessage}</p>
+      </article>
     }
 
     if(code >= UPLOADING) {
-      message = `Submission of ${file.name} currently in progess. You can drag another LAR file to this area, or click in the box to select a LAR file to upload..`
+      message = <article>
+        <p>Submission of <strong>{file.name}</strong> is currently in progess.</p>
+        <p className="file-selected">{howToMessage}</p>
+      </article>
+    }
+
+    if(code === SIGNED) {
+      message = <article>
+        <p>Your submission of <strong>{file.name}</strong> is complete.</p>
+        <p className="file-selected">{howToMessage}</p>
+      </article>
     }
   }
 
-  return <button onClick={e=>e.preventDefault()}>{message}</button>
+  return <span className="usa-button" role="button" onClick={e=>e.preventDefault()}>{message}</span>
```

Comment:
Because I added the `<article>` and `<p>` to the message and those aren't technically allowed within a `<button>`. It does render correctly with `<button>` though.